cumulus/pallets/collator-selection/Cargo.toml

```
@@ -6,7 +6,7 @@ homepage = "https://substrate.io"
 license = "Apache-2.0"
 name = "pallet-collator-selection"
 readme = "README.md"
-repository = "https://github.com/paritytech/cumulus/"
+repository = "https://github.com/paritytech/polkadot-sdk/"
```

```suggestion
repository.workspace = true
```

That will allow us to control it centrally.